index.html

```diff
@@ -699,6 +699,70 @@ <h2>
     const observer = new ComputePressureObserver(pressureChange);
     observer.observe("cpu");
   </pre>
+  <p>
+    In the following example, we want to demonstrate the usage of {{ComputePressureObserver/takeRecords()}},
+    by retrieving the remaining |entries| accumulated since the last callback.
+  </p>
+  <p>
+    It is recommended to do so before {{ComputePressureObserver/disconnect()}}, otherwise {{ComputePressureObserver/disconnect()}} will clear them and they will be lost.
+  </p>
+  <pre class="example js" title="Usage of takeRecords">
+    function pressureChange(entries) {
+      // do something with entries
+    }
+
+    const observer = new ComputePressureObserver(pressureChange);
+    observer.observe("cpu");
+
+    // observe.disconnect() will be called promptly, no time to wait for
+    // callback. latest entries are needed immediately or will be lost otherwise.
+    const entries = takeRecords();
+
+    // do something with entries.
+
+    observer.disconnect();
```

Comment:
Can you add a comment why we'd disconnect.

Reply:
Added in example description